unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/ISceneController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using DCL.Controllers;
+using DCL.Models;
+
+namespace DCL
+{
+    public delegate void OnOpenNFTDialogDelegate(string assetContractAddress, string tokenId, string comment);
```

Comment:
Not super happy with this delegate here, but it wouldnt make sene to tackle down this technical debt in this PR.

Reply:
If the interface needs it, it has to be here. Not sure of what we could do better in this case as moving it to another assembly seems overkill.

Reply:
Yeah but at the same time, it seems strange that the `ISceneController`  interface needs an `OpenNFTDialgo` delegate, doesnt it?